src/plugins/score-lib-process/Process/Script/ScriptEditor.cpp

```diff
@@ -2,12 +2,19 @@
 
 #include "MultiScriptEditor.hpp"
 #include "ScriptWidget.hpp"
+#include "score/tools/FileWatch.hpp"
```

Comment:
I try to always keep the < > include style instead of " " in ossia as much as possible, as it is less ambiguous